hermes-common/src/main/java/pl/allegro/tech/hermes/common/kafka/offset/PartitionOffsets.java

```diff
@@ -0,0 +1,26 @@
+package pl.allegro.tech.hermes.common.kafka.offset;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class PartitionOffsets implements Iterable<PartitionOffset> {
```

Comment:
is this the same class as  pl.allegro.tech.hermes.domain.subscription.offset.PartitionOffsets? i would rather move old structs to `domain` and `infra` packages than grow `common`